securing_apps/topics/oidc/keycloak-gatekeeper.adoc

```diff
@@ -146,6 +146,13 @@ bin/{generic_adapter_name} \
 
 By default the roles defined on a resource perform a logical `AND` so all roles specified must be present in the claims, this behavior can be altered by the `require-any-role` option, however, so as long as one role is present the permission is granted.
 
+==== OpenID Provider Communication
+By default the communication with the OpenID provider is direct. if you wish, you can specify a forwarding proxy server in your configuration file:
```

Comment:
The "i" in "if you wish" needs to be capitalized.

Reply:
Done :)

Reply:
Thank you.